lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -330,13 +335,16 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
         |> Enum.flat_map(
           &case fetch_stop_name_fn.(&1) do
             :error -> []
+            "Massachusetts Avenue" -> ["Mass. Ave"]
```

Comment:
I spoke to Hannah about this since she is handling the client work. We think it's better here because Mass Ave is indeed the only stop that is always abbreviated.